tests/unittests/conftest.py

```diff
@@ -55,3 +56,12 @@ def fake_filesystem(mocker, tmpdir):
             func = getattr(mod, f)
             trap_func = retarget_many_wrapper(str(tmpdir), nargs, func)
             mocker.patch.object(mod, f, trap_func)
+
+
+PYTEST_VERSION_TUPLE = tuple(map(int, pytest.__version__.split(".")))
+
+if PYTEST_VERSION_TUPLE < (3, 9, 0):
```

Comment:
confirmed I see the pytest commit https://github.com/pytest-dev/pytest/commit/ab3637d486 as defined/included in release 3.9.0. Thanks for the fix